models/readme.md

```diff
@@ -1 +1,164 @@
-The MLAgility model corpus will go here.
\ No newline at end of file
+# MLAgility Benchmark
+
+This directory contains the MLAgility benchmark, which is a large collection of models that can be evaluated using the [`benchit` CLI tool](https://github.com/groq/mlagility/blob/main/docs/benchit_user_guide.md).
+
+## Table of Contents
+
+- [Benchmark Organization](#benchmark-organization)
+- [Running the Benchmark](#running-the-benchmark)
+- [Model Template](#model-template)
+
+## Benchmark Organization
+
+The MLAgility benchmark is made up of several corpora of models (_corpora_ is the plural of _corpus_... we had to look it up too). Each corpus is named after the online repository that the models were sourced from. Each corpus gets its own subdirectory in the `models` directory. 
```

Comment:
Do we want to create a distinction between scripts and models (each script may instantiate multiple models)?

Reply:
Yes, added!